src/video_core/texture_cache/texture_cache.h

```diff
@@ -776,6 +787,44 @@ class TextureCache {
         }
     }
 
+    /**
+     * `GetNullSurface` gets a null surface based on a target texture.
+     * @param target, The target of the null surface.
+     **/
+    TView GetNullSurface(SurfaceTarget target) {
+        const u32 i_target = static_cast<u32>(target);
+        auto it = invalid_cache.find(i_target);
+        if (it != invalid_cache.end()) {
+            TSurface surface = it->second;
+            return surface->GetMainView();
```

Comment:
```suggestion
            return it->second->GetMainView();
```

This way you don't make an unnecessary surface copy.